python/ray/actor.py

```diff
@@ -534,6 +534,73 @@ def options(self, **actor_options):
         The arguments are the same as those that can be passed
         to :obj:`ray.remote`.
 
+        Args:
```

Comment:
is it possible to add them and generate with option_utils? 
For example https://github.com/ray-project/ray/blob/master/python/ray/_private/ray_option_utils.py

we have task only options, actor only options, common options, I feel maybe the comment should be put there.